tiantianXia/assignments/treesBFS/lc103/lc103.js

```diff
@@ -2,6 +2,39 @@
 
 // Given a binary tree, populate an array to represent its zigzag level order traversal. You should populate the values of all nodes of the first level from left to right, then right to left for the next level and keep alternating in the same manner for the following levels.
 
+/**
+ * input: root: root of the binary tree
```

Comment:
Great overview here. Could be helpful to also include a note note about edge cases here.